submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/integration/PersistenceContextImpl.kt

```diff
@@ -28,6 +29,10 @@ open class PersistenceContextImpl(
     private val subDbMapper: SubmissionDbMapper,
     private val subMapper: SubmissionMapper
 ) : PersistenceContext {
+    override fun createAccNoPatternSequence(pattern: AccPattern) {
+        sequenceRepository.save(Sequence(pattern.prefix))
```

Comment:
I believe we need to use both prefix and suffix here

Reply:
This won't make sense after I remove the suffix as discussed before.